gtda/homology/point_clouds.py

```diff
@@ -457,3 +457,194 @@ def transform(self, X, y=None):
         Xt = np.stack(Xt)
         Xt = np.nan_to_num(Xt, posinf=self.infinity_values_)
         return Xt
+
+
+@adapt_fit_transform_docs
+class EuclideanCechPersistence(BaseEstimator, TransformerMixin):
+    """`Persistence diagrams <https://giotto.ai/theory>`_ resulting from
+    `Cech filtrations <https://giotto.ai/theory>`_.
```

Comment:
it seems that, currently, Cech filtrations are not covered in the theory.

Reply:
I created an issue #295.